executor/executor_test.go

```diff
@@ -2090,10 +2090,10 @@ func (s *testContextOptionSuite) TestCoprocessorPriority(c *C) {
 	cli.priority = pb.CommandPri_Low
 	tk.MustQuery("select count(*) from t")
 
-	cli.priority = pb.CommandPri_Low
+	cli.priority = pb.CommandPri_Normal
```

Comment:
why we need change this?

Reply:
the logical to define whether a query is expensive has changed.

Reply:
Maybe we should insert more data in this test, to make it cover that full table scan is expensive thus have low priority.